client/components/edit/create/__snapshots__/CreateObject.test.tsx.snap

```diff
@@ -19,7 +19,7 @@ exports[`CreateObject renders appropriately with default settings 1`] = `
       name="parent"
       required=""
       type="text"
-      value=""
+      value="foo:1234"
```

Comment:
foo:1234 is not a default setting -- is this test passing more parameters than intended?

Reply:
Reverted change.

Reply:
In addition to reverting the change to the snapshot, I think it probably makes sense to pass `{}` instead of `props` to `CreateObject` in line 68 of CreateObject.test.ts (that may be a pre-existing bug... but the intent was to test how it behaves if you just rely on the default values provided by the function).

Reply:
Made change.